src/components/CardAdministration.vue

```diff
@@ -230,7 +230,7 @@ interface Props {
   assessments: Assessment[];
   showParams: boolean;
   isSuperAdmin: boolean;
-  creatorName: string;
+  creatorName?: string;
```

Comment:
The creatorName is not optional. Doesn't the `withDefaults` handle this? 

If there's any missing assignments, I can just write a script to update them.